src/ngx_http_lua_ssl_certby.c

```diff
@@ -880,6 +880,30 @@ ngx_http_lua_ffi_ssl_server_name(ngx_http_request_t *r, char **name,
 #endif
 }
 
+int
+ngx_http_lua_ffi_ssl_client_addr(ngx_http_request_t *r, char **addr,
```

Comment:
OK, you are returning a raw client address here, so you should rename this C function to something like `ngx_http_lua_ffi_ssl_raw_client_addr`, just like the existing `ngx_http_lua_ffi_ssl_raw_server_addr` C function. We should be consistent with ourselves here.

Reply:
I think we should do raw_client_addr here, which is more useful. Parsing the binary form is much more efficient than parsing the textual form.